src/app/shared-services/inferences/engine/simple-engine/index.ts

```diff
@@ -5,11 +5,11 @@ import {
   InferenceDefinition,
```

Comment:
Why is this file renamed to `index.ts` rather than staying with `simple-engine.ts`?

Reply:
agreed, I renamed it.